python_builtin_functions.py

```diff
@@ -4,86 +4,170 @@ def myabs(number):
         return -number
     return number
 
+
 def myall(iterable):
     """ Implementation of built-in `all` function """
-    pass
+    for item in iterable:
+        if not item:
+            return False
+    return True
 
 
 def myany(iterable):
     """ Implementation of built-in `any` function """
-    pass
+    for item in iterable:
+        if item:
+            return True
+    return False
 
 
 def mylen(s):
     """ Implementation of built-in `len` function """
-    pass
+    i=0
```

Comment:
You can rename `i` as `count` or `length`

Reply:
Changed.